### DISCUSSION TITLE
input system options

### DISCUSSION THREAD

**Discussion by charlesisfeline** [DEMOLITIONDON96/Demolition-Engine#89] [Ideas (Other)]
# Input System Options
title says it

Basically the rating system options, except its related to input

## list of input system options to be included

1. Psych Engine (the default)
2. OG FNF Pre-Week 5 (it sucks, but yeah it can be listed)
3. KE Current (for those who like kade engine better due to its own input system)
4. KE (VDAB/1.2) (for spammy songs)
5. FPS Plus (idk)

**Comment by ThatOneSillyGuyDon** [comment]
KE (current) Input system is already added as an option, maybe that option can be updated later on as a list of input systems you can choose from, like with the judgement skins and rating systems so far

**Reply by charlesisfeline** [reply]
ik
